task2/Comment.php

```diff
@@ -0,0 +1,23 @@
+<?php
+include "User.php";
+
+class Comment
+{
+    private User $user;
+    private string $message;
+    public function __construct($user, $message)
```

Comment:
Не указаны типы входных параметров
```suggestion
    public function __construct(User $user, string $message)
```